### DISCUSSION TITLE
Any plans to extend this to CyperArk EPM?

### DISCUSSION THREAD

**Discussion by A55imilat0r** [pspete/psPAS#530] [Ideas]
Any plans to extend this to CyperArk EPM?

**Comment by pspete** [comment]
No, this is focused on the privileged access security products, so scope is limited to what you access when authenticated to the password vault Web access or privilege cloud portal. 

Believe there are other tools available on GitHub with a focus on EPM.

**Reply by A55imilat0r** [reply]
Hi thanks for the quick response.

Yes I have found some on here for EPM but nothing as comprehensive as what you have done here for PAS. Nice work by the way.  

From what I can tell the AIPs via postman are consistent across all the CA products so our devs will be able to learn from what you have done here. 

Cheers
